### ISSUE TITLE
"or" not defined after second equal call with either

### ISSUE THREAD

**Issue by Malusia** [philbooth/check-types.js#56]
Hey,

I think I have found a bug, do something wrong or its an improvment ;-) Here is my case:

```javascript
var x = 'str-n';
check.assert.either.equal(x, 'str-1', 'Error msg.')
  .or.equal(x, 'str-2')
  .or.equal(x, 'str-3');
```

In this case `or` is not defined after the second `equal` call.

**Comment by philbooth** [comment]
Thanks for raising this, implementing it is quite tricky I think. I'll have a play around and see what I can come up with.

**Comment by philbooth** [comment]
I tried fixing this issue but failed. And after giving it some thought, I've decided to remove `either` altogether. So as of version `6.0.0`, there is no `either` modifier.

The batch function `any` works much better and for more cases, so having a less-good version of the same thing seemed unnecessary and wrong. And for simpler boolean logic, I think it reads better in plain JS. Removing `either` reduces the library size a little bit too, which is never a bad thing.
